ci/check_gucs_are_alphabetically_sorted.sh

```diff
@@ -0,0 +1,3 @@
+set -euo pipefail
+grep -o -E "(\.*\"citus.\w+\")," src/backend/distributed/shared_library_init.c > gucs.out
```

Comment:
Minor: You can add a comment like "extract citus gucs in the form of "citus.X", so that when someone else looks at this regex, they have an easier time understanding it.